action-scheduler.php

```diff
@@ -5,11 +5,10 @@
  * Description: A robust scheduling library for use in WordPress plugins.
  * Author: Prospress
  * Author URI: http://prospress.com/
- * Version: 1.6.0-dev
+ * Version: 2.0.0-dev
  * License: GPLv3
  *
- *
- * Copyright 2017 Prospress, Inc.  (email : freedoms@prospress.com)
+ * Copyright 2018 Prospress, Inc.  (email : freedoms@prospress.com)
```

Comment:
Question: Should this actually include the prior copyright years, rather than replacing them? e.g. `Copyright 2017-2018 Prospress, Inc.`?

Reply:
This isn't enough to prevent merge, so I'm going to go ahead. It's easy enough to update this later as needed.